tests/test_filters.py

```diff
@@ -577,7 +566,6 @@ def test_outline_composite(composite):
     assert isinstance(output, pyvista.PolyData)
     output = composite.outline(nested=True, progress_bar=True)
 
-    # vtk 9.0.0 returns polydata
```

Comment:
I can't suggest an edit for some reason, but it looks like this comment might still be relevant.

Reply:
(If you hover over the "suggest edit" icon it says "Applying suggestions on deleted lines is not supported.)

Reply:
(If you hover over the "suggest edit" icon it says "Applying suggestions on deleted lines is not supported.)

Reply:
I'm not sure if "modern vtk returns polydata" is better, but I'm trying to remove comments that relate only to older versions. Removing version info sort-of helps logically understand that this was older behaviour.

Reply:
I was assuming that versions >9.0.0 return MultiBlock rather than PolyData, but I don't actually know.  If this indeed is a difference between 8.x and 9.x, we should clean up this test better.  Either way, I think this comment is useful, either to describe why the test is the way it is or as a pointer for future cleanup.